recipes/gnuplot/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+opts="
+    --without-x \
+    --without-lua \
+    --without-latex \
+    --without-libcerf \
+    --with-qt4
+    "
+
+opts="$opts --with-readline=$PREFIX"
+
+LIBS="-liconv" ./configure --prefix=$PREFIX $opts
```

Comment:
Change to `LDFLAGS="$LDFLAGS -Wl,-rpath,$PREFIX/lib" LIBS="-liconv" ./configure --prefix=$PREFIX $opts`